advanced_communication_options/update_comms_settings.py

```diff
@@ -125,8 +196,8 @@ def check_pi3():
 	if hw_id=="BCM2709":
```

Comment:
Lol, this is what I get.

![pi3_version](https://user-images.githubusercontent.com/26958764/27441737-19565984-5777-11e7-80ed-eb86ccf20977.PNG)